### PR TITLE
Add helper functions to get stressmodels from pastastore

### PR DESCRIPTION
**Pull request by dbrakenhoff**
Make it even easier to build Pastas models! As suggested by @FransSchaars.

Build stressmodels from stored stress time series with `pstore.get_stressmodels()`

```python
sm = pstore.get_stressmodel("well1", rfunc=ps.Hantush, up=False)
```
Methods support nearest option when x and y-coordinates are stored for all stresses and oseries! Add them directly to pastas models with `pstore.add_stressmodel()`
```python
ml = ps.Model(oseries)  # or use `pstore.create_model(oseries_name)`
pstore.add_stressmodel(ml, stresses=["nearest prec", "nearest evap"], stressmodel=ps.RechargeModel)
```

\+ some other clean up and improvements

### CODE REVIEW

**Comment by dbrakenhoff** [comment]
Test fail is because i annoyed the KNMI api, so all good to go after review.

**Comment by martinvonk** [comment]
Maybe we can separate the deletion of the arctic and pystore to another pr such that @FransSchaars can review and test this?

File: pastastore/store.py [hunk 1]
```diff
@@ -813,50 +812,352 @@ def add_recharge(
...
+        return stressmodel(
+            **stresses,
+            rfunc=rfunc(**rfunc_kwargs),
+            name=stressmodel_name,
+            **kwargs,
+        )
+
+    def add_stressmodel(
+        self,
+        ml: ps.Model,
```
[the remark this answers is not in the document]
Comment (dbrakenhoff) [reply pastastore/store.py]: Yeah, we could add support for in-place model modifications. See new commit.

**Comment by dbrakenhoff** [comment]
> Maybe we can separate the deletion of the arctic and pystore to another pr such that @FransSchaars can review and test this?

Just ignore the deletions? It should work fine. (I'm a bit lazy and don't feel like separating the two :).)

**Pull request merged**